watcher/lib/influxdb/influxdb.go

```diff
@@ -27,22 +27,59 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+// Config is the configuration for InfluxDB
+type Config struct {
+	// InfluxDBAdminUser is the InfluxDB admin username
+	InfluxDBAdminUser string
```

Comment:
These prefixes are all redundant as the struct is already inside a packages called `influxdb`